backend/src/v4/middlewares/issue.js

```diff
@@ -16,14 +16,24 @@
  */
 
 "use strict";
+
 (() => {
 	const checkPermissions = require("./checkPermissions").checkPermissions;
 	const C	= require("../constants");
+	const { v5Path } = require("../../interop");
+	const { validateMany } = require(`${v5Path}/middleware/common`);
+	const { isModuleEnabled } = require(`${v5Path}/middleware/permissions/components/teamspaces`);
+	const { ADD_ONS_MODULES } = require(`${v5Path}/models/teamspaces.constants`);
+
+	const setTeamspaceParam = (req, res, next) => {
```

Comment:
There's something that does something similar in the middleware file already : https://github.com/3drepo/3drepo.io/blob/master/backend/src/v4/middlewares/middlewares.js#L165

we can probably just use it